config/bolt/contenttypes.yaml

```diff
@@ -139,10 +139,10 @@ entries:
     taxonomy: [ categories, tags, groups ]
     record_template: entry.twig
     listing_template: listing.twig
-    listing_records: 10
+    listing_records: 20
     default_status: published
     sort: -publishedAt
-    recordsperpage: 20
+    recordsperpage: 6
```

Comment:
Rename to records_per_page.
How this one differs from listing_records?

Reply:
records_per_page is the amount in the backend listing, listing_records is the amount on a default listing page in the frontend.. I'll make the names consistent, with a fallback for the old-school bolt 3 naming.

Reply:
I don't think we need a fallback here